spine-c/src/spine/Bone.c

```diff
@@ -209,9 +206,9 @@ float spBone_getWorldScaleY (spBone* self) {
 }
 
 void spBone_worldToLocal (spBone* self, float worldX, float worldY, float* localX, float* localY) {
-	float x = worldX - self->worldX, y = worldY - self->worldY;
 	float a = self->a, b = self->b, c = self->c, d = self->d;
 	float invDet = 1 / (a * d - b * c);
+	float x = worldX - self->worldX, y = worldY - self->worldY;
 	*localX = (x * d * invDet - y * b * invDet);
 	*localY = (y * a * invDet - x * c * invDet);
 }
```

Comment:
I think I saw some missing functions before and I assumed that these functions are used in Spine Editor only. Also I didn't find usages of these functions in the ported code. Should we port these functions to be used in other runtimes like spine-cocos2dx or by some another reason?